src/encord_active/lib/encord/actions.py

```diff
@@ -211,18 +231,56 @@ def create_project(
 
         return new_project
 
+    def fix_pickle_files(self, embedding_type: EmbeddingType, renaming_map):
+        def fix_pickle_file(up, renaming_map):
+            up["label_row"] = renaming_map[up["label_row"]]
+            up["data_unit"] = renaming_map[up["data_unit"]]
+            url_without_extension, extension = up["url"].split(".")
+            changed_parts = [renaming_map[x] if x in renaming_map else x for x in url_without_extension.split("/")]
+            up["url"] = "/".join(changed_parts) + "." + extension
+            return up
+
+        collection = load_collections(embedding_type, self.project_file_structure.embeddings)
+        updated_collection = [fix_pickle_file(up, renaming_map) for up in collection]
```

Comment:
[curious]🤔 

I'm not sure what `up` stands for, can you come up with something more meaningful?